test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -10,6 +10,7 @@
   > (library
   >  (name $lib)
   >  (private_modules bar)
+  >  (flags :standard -bs-jsx 3)
```

Comment:
why do we set this flag manually? isn't this already hard coded by this PR?

Reply:
it was used for previous attempts to connect flags from `dune` file into merlin directive. Removed in https://github.com/ocaml/dune/pull/6476/commits/858dac6666a97ce0b573c5dd8b6499175efd3480.